server/src/controllers/employer/addEmployers.js

```diff
@@ -0,0 +1,71 @@
+const logger = require("pino")();
+const Employer = require("../../models/employer.model");
+const {
+  addDefaultDates,
+  setOwnerAndCreator,
+} = require("../../utils/creation_util");
+
+const addEmployersRequestHandler = async (req, res) => {
```

Comment:
When creating an employer, allow the user to specify the date added. It may seem a bit odd, but we will be migrating from an old system to a new system so we want to preserve the addition date if possible.

Reply:
Will do. Should I still change the date updated to match now?

Reply:
It should be fine the way it is now